src/components/EthVideo.js

```diff
@@ -14,7 +17,11 @@ const EthVideo = ({ className, videoSrc }) => {
     <div className={className}>
       <video
         id="hero-video"
-        alt={`ETH glyph video - ${isDarkTheme ? "Dark" : "Light"}`}
+        alt={`${translateMessageId("comp-eth-video-alt", intl)} - ${
```

Comment:
While this implementation looks correct to me, @ryancreatescopy what are you thoughts on this? Having a different alt text for dark vs. light seems overkill to me. It's not something I'd want to to roll out on every image variation.

Reply:
I think this could just have a shared "Video of a transforming Eth glyph" alt description. No need to describe the mode.